scss/utilities/_position.scss

```diff
@@ -23,3 +23,8 @@
     z-index: $zindex-sticky;
   }
 }
+
+.pos-s { position: static; }
```

Comment:
You are right!
Could be a good naming scheme ?
`pos-sti`
`pos-sta`
`pos-rel`
`pos-abs`
`pos-fix`